### ISSUE TITLE
[vue-facing-decorator][vue-3][vue-i18n@9] bug to load local i18n bloc

### ISSUE THREAD

**Issue by rdhainaut** [intlify/bundle-tools#268]
### Reporting a bug?

I'm unable to use the <i18n> custom block in Single File Components (SFC) in a project that employs vue-facing-decorator, Vue 3, and Vite. The **local translations** are missing, with all i18n methods displaying the key instead of the expected message.

### Expected behavior

Load messages from the i18n block and display them when used in the template.

### Reproduction

repo with issue: https://github.com/rdhainaut/vue3-vfd-i18n-issue

WARNING:
This project utilizes class-based components along with [vue-facing-decorator](https://github.com/facing-dev/vue-facing-decorator). This is now the official recommendation for upgrading from Vue 2 class components/property decorators to Vue 3.

The issue arises only when using the recommended setting with reflect-metadata to set up the alias for vue-facing-decorator.
config: https://github.com/rdhainaut/vue3-vfd-i18n-issue/blob/master/vite.config.ts line: 18
doc: https://facing-dev.github.io/vue-facing-decorator/#/en/compatibility/reflect-metadata/reflect-metadata

### Issue Package

unplugin-vue-i18n

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (16) x64 11th Gen Intel(R) Core(TM) i9-11950H @ 2.60GHz
    Memory: 44.39 GB / 63.67 GB
  Binaries:
    Node: 18.12.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.19.3 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22621.1485.0)
```

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/intlify/bundle-tools/blob/main/.github/contributing.md).
- [X] Read the README
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/intlify/bundle-tools/discussions).

**Comment by rdhainaut** [comment]
Edit: I have found the problem of issue and submit a fix via a PR
More details here: https://github.com/rdhainaut/bundle-tools/commit/4cdd4449d48afe87e2533b20008582d9d4bb2db5

**Comment by kazupon** [comment]
Fixed in #269 
Thanks!
